### PR TITLE
Clickstream tutorial

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Adds Clickstream analysis tutorial (project template).

## Review

* [Page to review](https://quixdocsdev.blob.core.windows.net/pr228/tutorials/clickstream/overview.html)

### CODE REVIEW

File: docs/tutorials/clickstream/overview.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Clickstream analysis
+
+In this tutorial you learn about a project template that analyzes a clickstream in real time, and generates events based on behavior. You use Python, rather than kSQL to perform aggregations and implement event logic based on user behavior.
+
+This tutorial uses a [Quix template project](https://github.com/quixio/template-clickstream){target=_blank}.
```
Comment (SteveRosam) [inline docs/tutorials/clickstream/overview.md]: Maybe:

This tutorial uses the Quix clickstream template ?

File: docs/tutorials/clickstream/get-project.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+| Environment Variable (secret) | Service(s) | Description|
+|---|---|---|
+| `redis_host` | Data ingestion, Real-time dashboard, Data aggregation, Data enrichment, Event detection | Host URL for Redis Cloud database |
+| `redis_port` | Data ingestion, Real-time dashboard, Data aggregation, Data enrichment, Event detection | Port for the Redis Cloud database |
+| `redis_username` | Data ingestion, Real-time dashboard, Data aggregation, Data enrichment, Event detection | Username for the Redis Cloud database - typically `default`|
+| `redis_password` | Data ingestion, Real-time dashboard, Data aggregation, Data enrichment, Event detection | Password for the Redis Cloud database - found from Security section |
+| `bearer_token` | Webshop frontend | A [PAT](../../develop/authentication/personal-access-token.md) that the web app uses to authenticate the Streaming Reader and Streaming Writer APIs |
+
+You need to [create secrets](../../deploy/secrets-management.md) for these and then assign them to the appropriate [environment variables](../../deploy/environment-variables.md).
```
Comment (SteveRosam) [inline docs/tutorials/clickstream/get-project.md]: I'm confused about this.

tl;dr mentioning environment variables here is confusing me. These are all held in secrets,

When you fork a repo (into a clean Quix environment) and before you sync, you have to create the secrets (keys and values). To do so you go to `secrets management` -> new secret -> create key and enter value. And you're done.

Secrets and environment variables are consumed in the same way, i.e. via `os.environ["API_SECRET"]` but they are populated in very different ways.

Comment (tbedford) [reply docs/tutorials/clickstream/get-project.md]: > To do so you go to secrets management -> new secret -> create key and enter value. And you're done.

Are you? In the past I've had to allocate the created secret to the corresponding environment variable. The exception to this is where you happen to name your secrets the same as the environment variable. 

So usually what happens for me is I do the fork, and then I see a load of environment variables that have "missing secret", so you have to create the secret, and make sure the environment variable is assigned the secret. 

I'll have another look at this though, I obviously need to explain it better.

**Review by SteveRosam — APPROVED** [review]
